invenio_files_rest/tasks.py

```diff
@@ -92,7 +104,13 @@ def migrate_file(src_id, location_name, post_fixity_check=False):
 
 @shared_task(ignore_result=True)
 def remove_file_data(file_id, silent=True):
-    """Remove file instance and associated data."""
+    """Remove file instance and associated data.
+
+    :param file_id: The :class:`invenio_files_rest.models.FileInstance` ID.
+    :param silent: . (Default: ``True``)
```

Comment:
I added it because I think there's both uses cases 1) remove a file if possible (silent) 2) must remove a file (non silent), so I would leave it as is.